modules/apps/frontend-editor/frontend-editor-ckeditor-web/src/main/resources/META-INF/resources/ckeditor5_classic.jsp

```diff
@@ -0,0 +1,37 @@
+<%--
+/**
+ * SPDX-FileCopyrightText: (c) 2024 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+--%>
+
+<%@ include file="/init.jsp" %>
+
+<%
+String contents = (String)request.getAttribute(CKEditorConstants.ATTRIBUTE_NAMESPACE + ":contents");
+Map<String, Object> editorData = (Map<String, Object>)request.getAttribute(CKEditorConstants.ATTRIBUTE_NAMESPACE + ":data");
+String name = namespace + GetterUtil.getString((String)request.getAttribute(CKEditorConstants.ATTRIBUTE_NAMESPACE + ":name"));
+
+JSONObject editorConfigJSONObject = null;
+
+if (editorData != null) {
+	editorConfigJSONObject = (JSONObject)editorData.get("editorConfig");
+}
+
+if (contents != null) {
+	editorConfigJSONObject.put("initialData", contents);
```

Comment:
This kind of mapping of old API to new avoids friction, and allows easier migration of usages in DXP. Some API may not be so easily mapped, but we can deal with it as we start migrating existing usages.